python/pyspark/ml/recommendation.py

```diff
@@ -127,27 +127,31 @@ class ALS(JavaEstimator, HasCheckpointInterval, HasMaxIter, HasPredictionCol, Ha
                               "StorageLevel for ALS model factors. " +
                               "Default: 'MEMORY_AND_DISK'.",
                               typeConverter=TypeConverters.toString)
+    unknownStrategy = Param(Params._dummy(), "unknownStrategy",
+                            "strategy for dealing with unknown users or items at prediction " +
+                            "time. Supported values: 'nan', 'drop'.",
```

Comment:
This is an interesting one, as I've been doing that (for the storage params), but actually when you do `explainParams` it gets duplicated so looks weird. So I think it may be best left out?